src/web-capabilities.ts

```diff
@@ -16,18 +16,18 @@ export enum CapabilityState {
  * information.
  */
 export class WebCapabilities {
-  private static cpuInfo = getCpuInfo();
+  static numCores = CpuInfo.getNumLogicalCores();
```

Comment:
I'm also not sure if we need to store `numCores` here. For places in which we need the number of logical cores, could we just call `CpuInfo.getNumLogicalCores()` directly?

Reply:
One possibility is to have something like this:

```
 // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
    if (CpuInfo.getNumLogicalCores()! < 2) {
      return CapabilityState.NOT_CAPABLE;
    }
```

Reply:
It's probably because `CpuInfo.getNumLogicalCores()` might actually return undefined. (As an aside, [is it really possible for `navigator` to not exist?](https://github.com/webex/web-capabilities/blob/afc77098838c8c0541d3c8ad44bccae1d66d4cba/src/cpu-info.ts#L5))

We don't want to use `!` because that implies that we know it will never be undefined.

If we don't know the number of logical cores we should just return `CapabilityState.UNKNOWN`.